components/homepage/Aurora.tsx

```diff
@@ -1,23 +1,29 @@
 import { Box } from "@chakra-ui/react";
 
 interface AuroraProps {
+  hasBackground?: boolean;
   size: { width: string; height: string };
   pos: { top: string; left: string };
   color: string;
 }
 
-export const Aurora: React.FC<AuroraProps> = ({ color, pos, size }) => {
+export const Aurora: React.FC<AuroraProps> = ({
+  hasBackground,
+  color,
+  pos,
+  size,
+}) => {
```

Comment:
`hasBackground` prop name does not make sense here - since the aurora always has a background

just make the `zIndex` configurable here. when `-1` is not suitable - we can provide prop `zIndex="auto"`